src/migration.rs

```diff
@@ -55,18 +55,20 @@ impl Migration {
         let mut changes = self.changes.clone();
         for change in &mut changes {
             match change {
-                &mut CreateTable(ref mut t, ref mut cb) |
-                &mut CreateTableIfNotExists(ref mut t, ref mut cb) => {
+                &mut CreateTable(ref mut t, ref mut cb)
+                | &mut CreateTableIfNotExists(ref mut t, ref mut cb) => {
                     if t.meta.has_id {
                         t.add_column("id", types::primary());
                     }
 
                     cb(t); // Run the user code
                     let vec = t.make::<T>(false);
-                    s.push_str( &match change {
-                        CreateTable(_, _) => T::create_table(&t.meta.name()),
-                        CreateTableIfNotExists(_, _) => T::create_table_if_not_exists(&t.meta.name()),
-                        _ => unreachable!()
+
+                    let name = t.meta.name().clone();
```

Comment:
This change was made to remove a warning that the compiler said could turn into a hard error in the future. Apparently with the 2018 a few of the borrowing mechanics changed/ became stricter. Not 100% sure what changed but now we're just cloning the name in the scope before we re-borrow which means no UB can happen between the mutable borrow in the outer scope